libbeat/template/load.go

```diff
@@ -67,7 +67,7 @@ func (l *Loader) Load() error {
 
 		err = l.LoadTemplate(tmpl.GetName(), output)
 		if err != nil {
-			return fmt.Errorf("could not load template: %v", err)
+			return fmt.Errorf("could not load template: %v\nTemplate is:\n%s", err, output)
```

Comment:
I would prefer not to have any new lines in the error message if possible. In case this ends up in the logs somewhere it would mean it's on multiple log lines so it will end up in multiple (unrelated) events.

Reply:
@cyrilleverrier Could you change the above?

Reply:
@ruflin I changed to output the message in one single line of text.
I also verified it locally